README.md

```diff
@@ -78,7 +78,24 @@ Note that we ignore `.csv` via our `.gitignore`.
 
 ### R
 
-TODO
+#### First time?
+
+Pull the Docker image from Docker Hub:
+
+`docker pull aboghoss/bargraph`
+
+#### Then
+
+```
+docker run -it \
+  -v <DATA_DIRECTORY>:/data \
+  -v <OUTPUT_DIRECTORY>:/out_dir \
+   aboghoss/bargraph \
+   -d /data/<FILENAME> \
+   -o /out_dir
+   -n <OUTFILE_NAME>
+   -a <ANONYMIZE>
```

Comment:
Awesome work! This is a little confusing to me... could you give an example? I'm confused what to call my `<DATA_DIRECTORY>` when line 94 is harcoded to `/data/`.

Reply:
Yes I can add more instructions. Basically the `-v` tag mounts that directory to the container so it can read the files there and write to it otherwise the container will just write internally and you won't get the results (maybe there's a better way to do this with docker?)

Reply:
Cool @aboghossian and I think -v flag is way to go. As a docker noob, I am just confused what to put for <DATA_DIRECTORY>:data/

is that... mounting whatever i have called my data directory to a directory called "data"?

so like i could do

"squirrels:data/"

and that would be fine?

Reply:
yeah exactly